### PR TITLE
Fixed style to comply with latest standards

### PR DESCRIPTION
**Pull request by PSC-darla**
Per the meeting on blah blah ....

blah blah blah

We hope everyone is :smile: with the new look!

Tear it apart!

/cc @PSC-dan, please take a look!

### CODE REVIEW

File: HelloWorld/MainWindow.xaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         Title="MainWindow" Height="350" Width="525">
-    <Grid>
-        <TextBlock HorizontalAlignment="Center" VerticalAlignment="Center">Hullo werld!</TextBlock>
+    <Grid Background="LightBlue">
+        <TextBlock FontWeight="Bold" FontSize="24" HorizontalAlignment="Center" VerticalAlignment="Center">Hullo werld!</TextBlock>
```
Comment (PSC-dan) [inline HelloWorld/MainWindow.xaml]: wasn't this supposed to be ExtraBold?

File: HelloWorld/MainWindow.xaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         Title="MainWindow" Height="350" Width="525">
-    <Grid>
-        <TextBlock HorizontalAlignment="Center" VerticalAlignment="Center">Hullo werld!</TextBlock>
+    <Grid Background="LightBlue">
```
Comment (PSC-dan) [inline HelloWorld/MainWindow.xaml]: LightGreen was the look for 2015

File: HelloWorld/MainWindow.xaml (hunk 1 above)
Comment (PSC-darla) [inline HelloWorld/MainWindow.xaml]: wasn't decided, just an idea tossed around

File: HelloWorld/MainWindow.xaml (hunk 2 above)
Comment (PSC-darla) [inline HelloWorld/MainWindow.xaml]: ok i'll fix that

**Comment by PSC-dan** [comment]
Looks good to me :+1:

**Pull request merged**